### ISSUE TITLE
Allow omittance of view JS via pjax

### ISSUE THREAD

**Issue by Sammaye** [yiisoft/yii2#9320]
I had this problem recently and solved it by checking the headers to remove the JS however I thought it would be good to be able to switch this in the widget.

So my scenario: basically I have a listview of users, each user has a "favourite" button. I have placed the JS code within the view file `_user.php` that is used in the config of the `ListView`.

The problem: this code is included multiple times due to how pjax works (of course) meaning that it will make multiple AJAX calls to the favourite page etc (a.k.a bad).

To summarise: it would be good to be able to switch this functionality in the widget.

**Comment by dynasource** [comment]
I am trying to understand what you're saying. Why would you place JS code in a view that is included multiple times?

**Comment by SilverFire** [comment]
I'm closing the issue because the question is unclear and TS is inactive
